src/aof.c

```diff
@@ -1869,14 +1879,19 @@ void backgroundRewriteDoneHandler(int exitcode, int bysignal) {
             /* AOF enabled, replace the old fd with the new one. */
             oldfd = server.aof_fd;
             server.aof_fd = newfd;
-            if (server.aof_fsync == AOF_FSYNC_ALWAYS)
+            if (server.aof_fsync == AOF_FSYNC_ALWAYS) {
+                latencyStartMonitor(latency);
                 redis_fsync(newfd);
-            else if (server.aof_fsync == AOF_FSYNC_EVERYSEC)
+                latencyEndMonitor(latency);
+                latencyAddSampleIfNeeded("aof-fsync-rewrite-done",latency);
```

Comment:
i think this would be a better name:
```suggestion
                latencyAddSampleIfNeeded("aof-rewrite-done-fsync",latency);
```